easybuild/tools/modules.py

```diff
@@ -331,6 +331,16 @@ def __setattr__(self, name, value):
 
         return True
 
+    def __delattr__(self, name):
```

Comment:
should we indicate that this is deprecated since we should prefer ModuleLoadEnvironment.remove instead?

Reply:
good point, fixed in https://github.com/easybuilders/easybuild-framework/pull/4784/commits/1406bdd9922d71a11ca12fce1eca1b2587049bac